gemrb/core/Scriptable/Actor.cpp

```diff
@@ -7331,7 +7331,7 @@ void Actor::UpdateModalState(ieDword gameTime)
 	//actually, iwd2 has autosearch, also, this is useful for dayblindness
 	//apply the modal effect about every second (pst and iwds have round sizes that are not multiples of 15)
 	// FIXME: split dayblindness out of detect.spl and only run that each tick + simplify this check
-	if (InParty && core->HasFeature(GFFlags::AUTOSEARCH_HIDDEN) && (third || (roundFraction % core->Time.defaultTicksPerSec == 0))) {
+	if (InParty && core->HasFeature(GFFlags::AUTOSEARCH_HIDDEN) && third && (roundFraction % core->Time.defaultTicksPerSec == 0)) {
```

Comment:
This is now bad for non-3ed games; nothing will check for hidden doors.

Reply:
> This is now bad for non-3ed games; nothing will check for hidden doors.

Spell "find traps" and modal find traps of rogues did this.

Reply:
Okay, so be it, as long as it's not a cheat. ;)

Reply:
All the games but iwd2 had it. That's why the fixme is there, since we're doing two things at once; now the game flag is effectively useless.